doc/builders/fetchers.chapter.md

```diff
@@ -1,12 +1,21 @@
 # Fetchers {#chap-pkgs-fetchers}
 
 Building software with Nix often requires downloading source code and other files from the internet.
-`nixpkgs` provides *fetchers* for different protocols and services. Fetchers are functions that simplify downloading files.
+To this end, Nixpkgs provides *fetchers*: functions to obtain remote sources via various protocols and services.
 
-## Caveats {#chap-pkgs-fetchers-caveats}
+As opposed to built-in fetchers such as [`builtins.fetchTarball`][https://nixos.org/manual/nix/stable/language/builtins.html#builtins-fetchTarball], which return a store path, Nixpkgs fetchers create [fixed output derivations](https://nixos.org/manual/nix/stable/#fixed-output-drvs).
+This means that files are downloaded at build time instead of at evaluation time, and are kept in the Nix store.
```

Comment:
Or maybe even two columns:
- <code>Cache key: all fetcher arguments / `name` + `hash` arguments</code>
- <code>Invalidated by: ttl expiration, nuking nix cache, store gc / store gc</code>

Reply:
I decided not to mention the cache key at all, because it's protocol-specific for builtins and Nixpkgs. Also it would have made the table very cluttered. It should be clear enough as it is now, and the major aspects are now documented.